### ISSUE TITLE
Question: python domain references from markdown cell

### ISSUE THREAD

**Issue by d-chambers** [spatialaudio/nbsphinx#227]
So I saw [this page](nbsphinx.readthedocs.io/en/0.3.5/markdown-cells.html#Links-to-Domain-Objects) on the docs but in the example a link to a rst file is used rather than a python domain, like those mentioned [here](https://stackoverflow.com/a/22714510). Is it possible to reference a python domain like this in a notebook so the path to the rst file isn't needed?

**Comment by mgeier** [comment]
No, currently that's not possible. This feature might be added in the future. If you want, you can implement it!
